app/services/batch/prediction/create_job.rb

```diff
@@ -14,7 +14,15 @@ def initialize(prediction_job, bajor_client = Bajor::Client.new)
 
       def run
         begin
-          bajor_job_url = bajor_client.create_prediction_job(prediction_job.manifest_url)
+          subject_set_id = prediction_job.subject_set_id
+          context = Context
+            .where(active_subject_set_id: subject_set_id)
+            .or(Context.where(pool_subject_set_id: subject_set_id))
+            .order(Arel.sql("CASE WHEN active_subject_set_id = #{subject_set_id} THEN 0 ELSE 1 END"))
```

Comment:
Upon reflection, I think it would be more appropriate to _only_ search for the pool_subject_set_id.  That's how training+prediction jobs currently work, and seems like an improvement (less complex) to drop active_subject_set_id from the query.